spec/lib/diaspora_federation/entities/account_migration_spec.rb

```diff
@@ -1,21 +1,110 @@
 module DiasporaFederation
   describe Entities::AccountMigration do
-    let(:new_diaspora_id) { alice.diaspora_id }
-    let(:new_author_pkey) { alice.private_key }
-    let(:hash) {
-      Fabricate.attributes_for(:account_deletion_entity).merge(
-        profile: Fabricate(:profile_entity, author: new_diaspora_id)
-      )
-    }
+    let(:old_user) { Fabricate(:user) }
+    let(:new_user) { Fabricate(:user) }
+    let(:old_diaspora_id) { old_user.diaspora_id }
+    let(:new_diaspora_id) { new_user.diaspora_id }
+
     let(:data) {
       hash.tap {|hash|
         properties = described_class.new(hash).send(:enriched_properties)
         hash[:signature] = properties[:signature]
       }
     }
-    let(:signature_data) { "AccountMigration:#{hash[:author]}:#{new_diaspora_id}" }
+    let(:signature_data) { "AccountMigration:#{old_diaspora_id}:#{new_diaspora_id}" }
+    alias_method :string, :signature_data
```

Comment:
Personally I would just write `let(:string) { signature_data }`, because I think it looks nicer ;) But whatever, this works for me if you like this more :)